src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Performance/CSharpUseSearchValues.cs

```diff
@@ -90,6 +90,8 @@ protected override bool IsConstantByteOrCharArrayCreationSyntax(SemanticModel se
         // ConstString.ToCharArray()
         internal static bool IsConstantByteOrCharArrayCreationExpression(SemanticModel semanticModel, ExpressionSyntax expression, List<char>? values, out int length)
         {
+            const SyntaxKind CollectionExpressionSyntaxKind = (SyntaxKind)9076;
```

Comment:
This constant is necessary because the SDK being referenced doesn't have it defined? Can you add a comment?